src/language-css/parser-postcss.js

```diff
@@ -422,6 +471,8 @@ function parseNestedCSS(node, options) {
       }
 
       if (lowercasedName === "import") {
+        node.import = true;
+        delete node.filename;
```

Comment:
Why?

Reply:
filename is a mess after format, suggestion?

Reply:
using delete reduce performance, we should avoid doing this if we really do not need it

Reply:
Also, in general, we're trying not to modify the AST during the printing phase.

Reply:
@thorn0 it is okay, but we should reduce number of `delete` where is unnecessary

Reply:
@evilebottnawi How many imports do Less files usually include? I don't believe this specific line can have any real performance implications.

Reply:
@thorn0 Depends on the size of the application, this is my recommendation, nothing more